adal/tests/Test.ADAL.NET.UIAutomation/Infrastructure/CertificateHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Test.ADAL.NET.UIAutomation
+{
+    public static class CertificateHelper
```

Comment:
It would help reviewers if you mentioned that you've copied code over from an existing repo, as we wouldn't have to review this again. You can add comments to your own PR.

Reply:
@bgavrilMS good point, ill clearly indicate that next time. thanks